nautobot/ipam/tests/test_filters.py

```diff
@@ -59,6 +61,10 @@ class VRFTestCase(FilterTestCases.FilterTestCase, FilterTestCases.TenancyFilterT
     queryset = VRF.objects.all()
     filterset = VRFFilterSet
     tenancy_related_name = "vrfs"
+    q_filter = {
+        "name": "icontains",
+        "description": "icontains",
+    }
```

Comment:
To make sure I understand - this is being declared so that we *don't* test for the `rd` field that is otherwise part of the searchfilter? Why so?

Reply:
The test's is written to dynamically add a random string to the end of the attr which does not conform to the the data expected.